packages/hardhat-core/test/internal/hardhat-network/provider/modules/web3.ts

```diff
@@ -22,13 +22,14 @@ describe("Web3 module", function () {
       describe("web3_clientVersion", async function () {
         it("Should return the right value", async function () {
           const res = await this.provider.send("web3_clientVersion");
-          const expectedHardhatVersion =
+          const expectedEthereumJSVersion =
             /^HardhatNetwork\/.*\/@nomicfoundation\/ethereumjs-vm/;
-          const expectedEDRVersion = /^edr\/.*\/revm\/.*$/;
+          const expectedEDRVersion =
+            /^HardhatNetwork\/.*\/@nomicfoundation\/edr/;
           const expected =
-            this.edrProcess === undefined
-              ? expectedHardhatVersion
-              : expectedEDRVersion;
+            this.isEdr() === true
```

Comment:
```suggestion
            this.isEdr()
```

Reply:
Thanks, this was my original code, but the JS linter wanted to check for true 😅